packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -47,6 +52,47 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         return _data[_start:_end];
     }
 
+    function checkAccesses(Vm.AccountAccess[] memory accountAccesses) internal returns (bool) {
+        // Deploy the contract using create to get the deployed code
+        SphinxForkCheck forkCheck = new SphinxForkCheck();
+
+        // If there aren't two calls, (one to the deployment proxy, and another to deploy the contract)
+        // then return false.
+        if (accountAccesses.length < 2) {
+            return false;
+        }
```

Comment:
If/when a real user encounters this, they are basically guaranteed to hit this first check. I only do the ones below to be extra cautious.